pages.pt_BR/linux/acountry.md

```diff
@@ -0,0 +1,16 @@
+# acountry
+
+> Imprime o país onde um endereço IPv4 ou nome do servidor estão localizados.
+> Mais informação: <https://manned.org/acountry>.
+
+- Imprime o país onde um endereço IPv4 ou nome do servidor estão localizados:
+
+`acountry {{examplo.com}}`
```

Comment:
```suggestion
- Imprima um país onde um endereço IPv4 ou host está localizado:

`acountry {{examplo.com}}`
```

Optional fix (error source: original page): unexpected `the` term, `a` expected.
Note: `Print a country where an IPv4 address or host is located:` can be used a translation source.